wherehows-web/app/styles/components/dataset-compliance/_compliance-table.scss

```diff
@@ -135,7 +135,6 @@
 
   &__guided-modal {
     @include nacho-container;
-    overflow: hidden;
```

Comment:
It hides the tooltip on purge key that we're trying to add. Since the purge key note is already on the edge, the tooltip on hover will be cut off. When removed it's not affecting anything so I felt like it was a good solution.